setupJest.js

```diff
@@ -8,7 +8,34 @@ require('zone.js/dist/sync-test');
 require('zone.js/dist/async-test');
 require('zone.js/dist/fake-async-test');
 require('jest-zone-patch');
+
 const AngularSnapshotSerializer = require('./AngularSnapshotSerializer');
+
+const MySerializer = {
```

Comment:
Can you rename this serializer to a more meaningful name ? I think the name "MySerializer" doesn't say what it does to improve the current AngularSnapshotSerializer

Reply:
Of course, I will. 

This PR is just to have some discussion about how this should be done. @thymikee does not have time for this probably, so this PR is just waiting :(.